canonical-web-and-design/charmhub.io
# Build about page

## Done

- Build about page
- Update main nav to highlight `About` when you're on any of the `about` sub-pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045/about
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e985987eff10a5291f09cfe) and [copy](https://docs.google.com/document/d/1UuNVajNHcWcF9d-fSnPFl8Zi4sDl21G5K-F09F8c3i8)

_Note the images are placeholders for now, and I kept the suru header_


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1255


## Scrennshot

![image](https://user-images.githubusercontent.com/40214246/82558747-3e280100-9b66-11ea-94bb-d9b1ab2ff1ac.png)
